price-feeder/CHANGELOG.md

```diff
@@ -44,11 +44,19 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-## [Unreleased]
+## Unreleased
+
+## Improvements
+
+- [1623](https://github.com/umee-network/umee/pull/1623) Update Umee version v3.1.0->v3.2.0
```

Comment:
This is necessary to comment on because price feeder's goreleaser would break if v3.2.0 was included in the next PF release

Until #1591 is fixed

Reply:
Usually, we don't put dependencies updates in a changelog.

Reply:
i'm ok w removing it 
https://github.com/umee-network/umee/pull/1636/commits/45f8c489b399488aa9e5093f160c18029bbf45b5